spiderable_server.js

```diff
@@ -16,6 +16,13 @@ var crypto = Npm.require('crypto');
 Spiderable.userAgentRegExps = [
     /^facebookexternalhit/i, /^linkedinbot/i, /^twitterbot/i];
```

Comment:
I'd match case, since it is easier to read, even if we ignore case.
On Jul 7, 2015 11:15 AM, "dr.dimitru" <notifications@github.com> wrote:

> In spiderable_server.js
> <https://github.com/jazeee/jazeee-meteor-spiderable/pull/3#discussion_r34058602>
> :
>
> > @@ -16,6 +16,13 @@ var crypto = Npm.require('crypto');
> >  Spiderable.userAgentRegExps = [
> >      /^facebookexternalhit/i, /^linkedinbot/i, /^twitterbot/i];
>
> Also in code I'll make them lowercase as we ignore case anyways, ok?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jazeee/jazeee-meteor-spiderable/pull/3/files#r34058602>
> .
>